test/csuite/schema_abort/main.c

```diff
@@ -312,6 +311,20 @@ test_cursor(THREAD_DATA *td)
     testutil_check(session->close(session, NULL));
 }
 
+/*
+ * set_flush_tier_delay --
+ *     Set up a random delay for the next flush_tier.
+ */
+static void
+set_flush_tier_delay(WT_RAND_STATE *rnd)
+{
+    /*
+     * We are checkpointing constantly, and we'll do a flush tier with a random delay between 0 -
+     * 10000 microseconds.
+     */
+    opts->tiered_flush_interval_us = __wt_random(rnd) % 10001;
+}
```

Comment:
This comment is not correct.  This program randomly sleeps either 0,1,2,3 seconds between each checkpoint.  So we might want to randomly wait between 0 and 10,000,000us between flush_tier calls.  Given that the program randomly aborts between 10 and 40 seconds after starting, that should usually give us at least one flush_tier and probably several before the forced abort.

Reply:
Updated the comment and tiered_flush_interval_us to randomly wait between 0 and 10,000,000us.